modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -174,5 +188,23 @@ def enum_zones
       logger.debug "Enumerated authoritative dns zones: #{zones}"
       zones
     end
+
+    def enum_records(zone_name, node_name, type)
+      records = []
+      response = execute "/EnumRecords #{zone_name} #{node_name}. /Type #{type}", "EnumRecords", true
+      response.each do |line|
+        line.chomp!
+        logger.debug "Extracting record from dnscmd output '#{line}'"
+        /^@?\s+\d+\s+(A|PTR)+\s+(?<record>\S+)/ =~ line
```

Comment:
@witlessbird I changed `[[:upper:]]` to `(A|PTR)` because of the change from `^@\s` to `^@?\s`
Now that `@` is optional, I think the regex needs to be a bit tighter elsewhere.